integrationTesting/scriptFixtures.go

```diff
@@ -0,0 +1,57 @@
+package integrationTesting
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type ScriptFixture struct {
+	db *sql.DB
+}
+
+func CreateLuaScriptFixture(db *sql.DB) ScriptFixture {
+	execSQL(db, "CREATE SCHEMA TEST")
+	execSQL(db, `
+CREATE LUA SET SCRIPT test.my_script (a DOUBLE)
```

Comment:
I would consider introducing a function which `lstrips` each line in a multi line strings so something like this would be possible:

```go
    stmt := clean(`
      CREATE LUA SET SCRIPT test.my_script (a DOUBLE)
      RETURNS DOUBLE AS
      function run(ctx) return 1 end
    `)
```

this would make the tests way easier to read.